packages/components/src/Actions/Action/Action.test.js

```diff
@@ -74,3 +74,57 @@ describe('Action', () => {
 		expect(wrapper.getNode().props.label).toBe('hello world');
 	});
 });
+
+describe('#wrapOnClick', () => {
+	it('should return onclick', () => {
+		// given
+		const onClick = jest.fn();
+		const eventFn = wrapOnClick({
+			onClick,
+			bsStyle: 'bsStyle',
+			inProgress: 'inProgress',
+			disabled: 'disabled',
+			hideLabel: 'hideLabel',
+			link: 'link',
+			onMouseDown: 'onMouseDown',
+			tooltipPlacement: 'tooltipPlacement',
+			tooltip: 'tooltip',
+			tooltipLabel: 'tooltipLabel',
+			available: 'available',
+			label: 'label',
+			model: {
+				id: '#model',
+			},
+			otherProperty: 'otherProperty',
+		});
+
+		// when
+		eventFn({
+			type: 'click',
+		});
+
+		// then
+		expect(onClick).toHaveBeenCalledWith(
+			{
+				type: 'click',
+			},
+			{
+				action: {
+					available: 'available',
+					bsStyle: 'bsStyle',
+					disabled: 'disabled',
+					hideLabel: 'hideLabel',
+					inProgress: 'inProgress',
+					label: 'label',
+					link: 'link',
+					onMouseDown: 'onMouseDown',
+					otherProperty: 'otherProperty',
+					tooltip: 'tooltip',
+					tooltipLabel: 'tooltipLabel',
+					tooltipPlacement: 'tooltipPlacement',
+				},
+				model: { id: '#model' },
```

Comment:
The entire action object is duplicated. Could you please put it all into a single object?